wit-definitions/types/wit/chain.wit

```diff
@@ -1,5 +1,6 @@
 interface chain {
     type chain-name = string;
+    type tx-hash = list<u8>;
```

Comment:
should this be bytes or a string? not a blocker, we can start with whatever and update later if we hit a problem, but I do think people are used to working with it as a string in indexers etc.